release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -1438,6 +1467,22 @@ In the following tables, features are marked with the following statuses:
 |Technology Preview
 |Technology Preview
 
+|Metrics Server
```

Comment:
Just wanted to make sure you planned on this being in the PR.

Reply:
Nope! I think it grabbed it on a rebase. Thanks!